webapp/src/test/java/com/box/l10n/mojito/android/strings/XmlReaderTest.java

```diff
@@ -0,0 +1,152 @@
+package com.box.l10n.mojito.android.strings;
+
+import com.google.common.io.Resources;
+import org.junit.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.xml.sax.SAXException;
+
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.util.List;
+
+import static com.box.l10n.mojito.android.strings.XmlHelper.*;
+import static org.junit.Assert.*;
+
+public class XmlReaderTest {
+
+    @Test
```

Comment:
can you add a test with all plural forms? and with a single plural form: "other"?